### DISCUSSION TITLE
Change import text from 'beatmaps' to 'beatmapsets'

### DISCUSSION THREAD

**Discussion by Joppe27** [ppy/osu#14180] [Ideas] [answered]
A very simple change that clears up some confusion.

Stable uses 'beatmaps' to describe the total amount of beatmaps/difficulties:
![stablenotif](https://user-images.githubusercontent.com/56885706/128692728-95e6a6e4-b3c1-40e2-b82d-003081eacf14.jpg)
Lazer uses 'beatmaps' to describe the total amount of beatmap*sets*:
![lazernotif](https://user-images.githubusercontent.com/56885706/128693154-4761ff03-dd29-4f86-9db0-d1d80d32ebdc.jpg)

This difference in wording can lead users to believe that not all beatmaps were imported correctly. This is of course not the case.

**Comment by peppy** [comment, marked as answer]
We don't use the term "beatmap set" anywhere outside of code. It is correct how it is.
